automatic/git.portable/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,11 @@
+$ErrorActionPreference = 'Stop'
+
+$installLocation = Join-Path $(Get-ToolsLocation) 'git'
```

Comment:
I do sometning like:

```powershell
$pp = Get-PackageParameters
$installDir = Get-ToolsLocation
if ($pp.InstallDir ) { $InstallDir = $pp.InstallDir}
Write-Host "$packageName is going to be installed in '$installDir'"
```

Using `Get-ToolsLocation` requires

```xml
    <dependencies>
      <dependency id="chocolatey" version="0.9.10" />
    </dependencies>
```

However, this becomes releveant **ONLY when choco#797 is implemented**. Until then people should not use this parameter as it will not be remembered when updating.

Now the PATH becomes problematic as if user changes the dir you don't know where is it installed later unless you remember it somewhere. You would have to remember the argument somewhere, such as in tools directory, and read it on uninstall.

So, I wouldn't do that now. `Get-ToolsLocation` is OK (just add dependency) and once 797 is up we can fix this and other stuff by adding `InstallDir`.